btrack/napari/writer.py

```diff
@@ -0,0 +1,30 @@
+"""
+This module is a writer plugin to export Tracks layers using BTrack
+"""
+from typing import Optional
+
+import numpy.typing as npt
+
+from btrack.io import HDF5FileHandler
+from btrack.utils import napari_to_tracks
+
+
+def export_to_hdf(
+    path: str,
+    data: npt.ArrayLike,
+    meta: dict,
+) -> Optional[str]:
+    tracks = napari_to_tracks(
+        data=data,
+        properties=meta.get("properties"),
+        graph=meta.get("graph"),
```

Comment:
Why `.get(` rather than `[`?

Reply:
it's not guaranteed that `properties` and `graph` exist. They are both dictionaries, so I've updated it that now so that if they don't exist an empty `dict` is created